Makefile

```diff
@@ -158,12 +158,12 @@ cluster-create::
 
 .PHONY: dev-install
 dev-install::
-	$(SCRIPTS_DIR)/fetch kind 0.17.0
+	$(SCRIPTS_DIR)/fetch kind 0.20.0
 	$(SCRIPTS_DIR)/fetch kubectl $(K8S_VERSION) # Install kubectl AFTER envtest because envtest includes its own kubectl binary
 
 .PHONY: test-e2e-teardown
 test-e2e-teardown:
-	$(SCRIPTS_DIR)/fetch kind 0.17.0
+	$(SCRIPTS_DIR)/fetch kind 0.20.0
```

Comment:
I believe the default node image used with kind 0.20.0 is based on 1.27.3 - can we update wherever necessary to set the appropriate node image?

Reply:
@everettraven I'm not sure how testing works in this project, I was just trying to keep things updated and working on any of my systems. But tests do not work for me, so that's why I raised this PR, it would be great if someone could kick those off. I can make this version whatever it needs to be.

Reply:
The `K8S_VERSION` in the export makes sure that the proper node image is pulled, if that's the question.